src/app/get-involved/donate-blood/TabSection/index.tsx

```diff
@@ -58,49 +86,70 @@ export default function TabSection() {
                         Centres in Biratnagar, Pokhara, Nepalgunj,
                         and Chitwan, or the nearest District Blood Bank or Hospital unit.
                     </p>
+                    <Table
+                        data={outsideValleyTable}
+                        columns={column}
+                        keySelector={(item) => item.id}
+                    />
                 </div>
             </TabPanel>
             <TabPanel className={styles.tabPanel} name="can">
                 <Heading>
                     Donating person has to apply:
                 </Heading>
-                <p>
-                    - be 18 to 60 years old
-                    - weight above 45 kg
-                    - have hemoglobin above 12 gm/dl
-                    - have blood pressure 110-160 / 70-96 mmHg
-                    - not to be pregnant, breastfeeding, and have at least
-                    8 days since the start of the recent menstruation
-                    - not having recent use of drugs or strong medicines
-                    (people who take strong medicine for a short period will
-                    not able to donate blood from one week up to 2 years)
-                    - not to had a medical surgery for 2 years
-                    If you have had one of these conditions, you are
-                    unfortunately restricted to donating blood but
-                    recommended to encourage your family members, friends and loved ones to donate:
-                    - cancer, heart diseases, HIV/AIDS, hepatitis B or C,
-                    hemophilia and thalassemia, diabetes, liver diseases,
-                    Polycythemia Vera, asthma, an endocrine disorder or a hormonal disorder
-                </p>
+                <ul className={styles.list}>
```

Comment:
Maybe use <ArticleBody /> for this and have the data in md for styling consistencies.